merkletree/pad.go

```diff
@@ -110,17 +124,23 @@ func (pad *PAD) LookupInEpoch(name string, epoch uint64) (*AuthenticationPath, e
 	return ap, nil
 }
 
+// GetSTR returns the signed tree root of the requested epoch.
+// This signed tree root is read from the snapshot array of the PAD.
+// It returns nil if the signed tree root has been removed from the memory.
 func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 	if epoch >= pad.latestSTR.Epoch {
 		return pad.latestSTR
 	}
 	return pad.snapshots[epoch]
 }
 
+// LatestSTR returns the most recent version of the signed tree root.
 func (pad *PAD) LatestSTR() *SignedTreeRoot {
 	return pad.latestSTR
 }
 
+// TB issues a new promise for the new binding and
```

Comment:
It may not be clear to a developer reading this what "the new binding" is, so I would make clear that this binding consists of the given name and value params.